cypress/integration/login.js

```diff
@@ -21,4 +21,10 @@ describe('Login functionality', function() {
         cy.get('.user-info__title').contains('TestCon User 9')
         cy.get('.main-nav').find('li').should('have.length', 2)
     })
+
+    it('Validate what date is selected as today', function() {
+
+        cy.get('.calendar--selected').should('have.class', 'calendar--today')
```

Comment:
You should validate the actual date, like today is 13th, tomorrow it will be 14th.

Reply:
Fixed